src/utils/PublishSnsMessage.ts

```diff
@@ -7,7 +7,7 @@ AWS.config.update({
 });
 
 const sns = new AWS.SNS();
-function publishMessage(sessionId) {
+export function publishMessage(sessionId: string) {
```

Comment:
publish message type is wrong.. it doesn't just take session id. it actually takes in a message object with a more involved structure. 

the message object has: [action, id, patch_session] keys as of now. you can put this in types. 

i am getting compilation error when i'm trying out on local

Reply:
resolved now